app/routes/__frontend/resources/virtual-coffee-handbook/get-involved/paths-to-leadership.mdx

```diff
@@ -99,19 +91,14 @@ This role gives members a chance to review the conversations during Coffee.
 - Organizing those notes into a clear and readable format.
 - Sharing the notes in Slack after the meeting is complete.
 
-#### Role Qualifications:
-
-- Attended at least 4 Coffees.
-- Reviewed the official Note Taker documentation.
-
 #### Role Benefits:
 
 - Active listening practice.
 - Efficient summarization skills.
 
 #### Working Conditions:
 
-- Approximately ~3 hours per week.
+- Approximately ~2 hours per week.
```

Comment:
We can decrease this to 2 hours per month.

Reply:
Made the change @BekahHW! :)